recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/kernel-module-gles/kernel-module-gles_1.9.bbappend

```diff
@@ -0,0 +1,8 @@
+require inc/xt_shared_env.inc
+
+EXTRA_OEMAKE += "PVRSRV_VZ_NUM_OSID=${XT_PVR_NUM_OSID}"
+
+PVRKM_URL = "git://git@gitpct.epam.com/epmd-aepr/pvr_km_vgpu_img.git"
```

Comment:
I think you are using the same value as in other patch.

Reply:
I have updated the .bb file, so these are two different now